modules/ide/xsk-ide-xsaccess/src/main/resources/META-INF/dirigible/xsk-ide-xsaccess/editor/editor.html

```diff
@@ -0,0 +1,342 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="page">
+
+  <head>
+    <title>.XSACCESS form-based editor</title>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="icon" sizes="any" href="data:;base64,iVBORw0KGgo=" />
+
+    <link type="text/css" rel="stylesheet" href="/services/v4/js/theme/resources.js/bootstrap.min.css" />
+    <link type="text/css" rel="stylesheet" href="/webjars/fontawesome/4.7.0/css/font-awesome.min.css" />
+
+    <!-- Custom IDE Styles -->
+    <link type="text/css" rel="stylesheet" href="/services/v4/js/theme/resources.js/ide.css" />
+
+    <script type="text/javascript" src="/webjars/jquery/3.6.0/jquery.min.js"></script>
+    <script type="text/javascript" src="/webjars/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+    <script type="text/javascript" src="/webjars/angularjs/1.8.2/angular.min.js"></script>
+    <script type="text/javascript" src="/webjars/angularjs/1.8.2/angular-resource.min.js"></script>
+    <script type="text/javascript" src="/services/v4/web/ide-core/ui/message-hub.js"></script>
+  </head>
+
+  <body ng-controller="PageController">
+    <div class="container">
+      <div class="page-header">
+        <h1>XS Access Description: {{file}}</h1>
+      </div>
+      <form>
+        <div class="form-group page-header">
+          <h2>Exposed</h2>
+          <div>
+            <input type="radio" class="form-group" ng-model="data.exposed" ng-value="false" id="dataExposedF" />
```

Comment:
`id="dataExposedF"` is not a good name. 
Identifiers and everything describing a name of something should not be shortened. Please, check all your code and remove any usages of shortened names. 
Other examples of bad naming are `custhdrsSAMEORIGIN`, `etagsF`, `etagsT`, `dataExposedACF`, `dataExposedACT`, etc. 
Also, use `camelCase` accordingly. For example, `customheaders` should be `customHeaders`

Reply:
OK